src/plugins/service-wispstock/plugin.ts

```diff
@@ -64,6 +65,8 @@ export class Plugin
   initAfterPlugins?: string[] | undefined;
   runBeforePlugins?: string[] | undefined;
   runAfterPlugins?: string[] | undefined;
+  public get Config() { return this.config; }
+  public get pcwd(): string { return this.pluginCwd; }
```

Comment:
Shouldn't we change this property name to match the get above for Config? `PluginCWD`?